Aptoide/aptoide-client-v8
# [Feature/ASV-1067] AppNext Metrics

**What does this PR do?**

   Adds more detail in similar apps metrics, specifically Similar_App_Interact and App_Viewed_Opened_From. More info about these metrics on the ticket.

Note that it's now using a specific analytics module for similar apps. This might not be desirable.

P.S. Who the hell named **App_Viewed**_Opened_From?

**Database changed?**

   No

**Where should the reviewer start?**

- AppViewSimilarAppAnalytics.java

**How should this be manually tested?**

  Verify that both events pass the correct parameters defined in the ticket. Check all the different possibilities, namely:
   - Similar Apps impression & click (with an aptoide ad, with an appnext ad and no ad)
   - AppView_Opened_From with any similar app.

 This can be done by checking logcat and Charles

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1067](https://aptoide.atlassian.net/browse/ASV-1067)


**Code Review Checklist**
